thorlcr/activities/diskread/thdiskreadslave.cpp

```diff
@@ -466,7 +468,6 @@ class CDiskReadSlaveActivity : public CDiskReadSlaveActivityRecord
             initMetaInfo(cachedMetaInfo);
             cachedMetaInfo.isSource = true;
             getPartsMetaInfo(cachedMetaInfo, partDescs.ordinality(), partDescs.getArray(), partHandler);
-            cachedMetaInfo.fastThrough = true;
```

Comment:
I think this means it will never be set - as far as I can see fastThrough is not set in getPartsMetaInfo

Reply:
@ghalliday - you're right - I didn't notice because it worked without checking if there was 1 part per slave, because in that case calcMetaInfoSize() called from getPartsMetaInfo() set everything to the meta of the single part.

I've now update with a check inside calcMetaInfoSize() which sets fastThrough if all parts are consistent.
Ideally it would do this for some other ThorDataLinkMetaInfo property values in general, but the properties are handled specifically elsewhere and not per part.